### PR TITLE
Run OpenDrift drift model based on simulation output saved in nc-files.

### PR DESCRIPTION
**Pull request by setmar**

### CODE REVIEW

**Comment by setmar** [comment]
[Advection using OpenDrift](https://trello.com/c/mfxPM1TO/10-advection-using-opendrift)

**Comment by havahol** [comment]
Could not run the notebook. 
I created a new netcdf file based on some arbitrary parameters and CTCS, but eventually got a "urcrnrlat must be between -90.000000 and 90.000000 degrees" error.

Can you share the netcdf input? Either by some smart way of sharing the netcdf itself (I know its coming, though), or by adding a cell which can be ran if the input file do not already exist?

File: python/SimpleOpenDriftRunner.ipynb [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+    "\n",
+    "u = np.array(ncfile.variables['hu'])\n",
+    "v = np.array(ncfile.variables['hv'])\n",
+    "x = np.array(ncfile.variables['x'])\n",
+    "y = np.array(ncfile.variables['y'])\n",
+    "t = np.array(ncfile.variables['time'])\n",
+    "\n",
+    "ncfile.close()\n",
+    "\n",
+    "w = np.sqrt(u*u+v*v)\n",
```
Comment (havahol) [inline python/SimpleOpenDriftRunner.ipynb]: Also, this thing fails with staggered grid, since u.shape != v.shape... :/

**Comment by setmar** [comment]
Having ignore_ghostcells set to True, and supplying offset_x- and offset_y-values that produce valid and 'reasonable' lonlat-values for OpenDrift should solve the mentioned issues. Only tested with CTCS. Added comment in notebook concerning this in the latest commit. We need to do some refactoring here at some point, to make it more general.

**Pull request merged**